tests/zfs-tests/tests/Makefile.am

```diff
@@ -2063,6 +2063,9 @@ nobase_dist_datadir_zfs_tests_tests_SCRIPTS += \
 	functional/zvol/zvol_stress/cleanup.ksh \
 	functional/zvol/zvol_stress/setup.ksh \
 	functional/zvol/zvol_stress/zvol_stress.ksh \
+	functional/dnode_stress/cleanup.ksh \
+	functional/dnode_stress/setup.ksh \
+	functional/dnode_stress/dnode_stress.ksh \
```

Comment:
I'd suggest renaming this something like `cp_stress.sh` and placing it in under `tests/zfs-tests/tests/functional/cp_files/`.